lr-cli/cortx_setup/commands/cluster/create.py

```diff
@@ -195,9 +195,6 @@ def run(self, **kwargs):
             )
             PillarSync().run()
 
-            self.logger.debug("Refreshing machine id on the system")
```

Comment:
Please update empty cluster validation in `cluster_id.py` along with the same PR.

Reply:
done